demisto_sdk/commands/prepare_content/preparers/marketplace_suffix_preparer.py

```diff
@@ -1,9 +1,14 @@
 import logging
+from typing import Dict
 
 from demisto_sdk.commands.common.constants import MarketplaceVersions
 
 logger = logging.getLogger("demisto-sdk")
 
+MARKETPLACE_TO_SUFFIX: Dict[MarketplaceVersions, str] = {
```

Comment:
Why move it outside the class?
It can be in the class, outside of the `prepare` method.

Reply:
doesn't really matter, moving it back in.